src/transactions/test/InvokeHostFunctionTests.cpp

```diff
@@ -65,10 +65,39 @@ makeSymbol(std::string const& str)
     return val;
 }
 
+static void
+submitOpToCreateContract(Application& app, Operation const& op,
+                         Bytes const& contract, Hash const& contractID,
+                         SCVal const& wasmKey, bool expectSuccess,
+                         bool expectEntry)
+{
+    // submit operation
+    auto root = TestAccount::createRoot(app);
+    auto tx = transactionFrameFromOps(app.getNetworkID(), root, {op}, {});
+    LedgerTxn ltx(app.getLedgerTxnRoot());
+    TransactionMeta txm(2);
+    REQUIRE(tx->checkValid(ltx, 0, 0, 0));
+    REQUIRE(tx->apply(app, ltx, txm) == expectSuccess);
+    ltx.commit();
+
+    auto contractCodeObj =
+        makeContract(contract.vec.begin(), contract.vec.end());
+    // verify contract code is correct
+    LedgerTxn ltx2(app.getLedgerTxnRoot());
+    auto ltxe2 = loadContractData(ltx2, contractID, wasmKey);
+    REQUIRE((bool)ltxe2 == expectEntry);
```

Comment:
nit: Please use `static_cast` - c-style casts should normally be avoided in c++.

Reply:
Done